data/hails.txt

```diff
@@ -1009,7 +1009,9 @@ phrase "friendly civilian"
 		"Some crazy militia officer "
 		"Walking into the spaceport, someone "
 	word
-		"asked me if I would help her fight local pirates. That's too dangerous for me!"
+		"asked me if I would help them fight local pirates. That's too dangerous for me!"
```

Comment:
Using "them" in the context of explicitly referring to a specific person seems awkward. A smoother way to avoid gendering the statement would be to remove the pronoun entirely, e.g. "Asked me to help fight local pirates."